### PR TITLE
add more variables from CAN Scraper state providers

### PR DESCRIPTION
**Pull request by TomGoBravo**
Part of https://trello.com/c/7HrFDkv7/773-import-ccd20-state-county-dashboard-data-into-combineddatasets

This adds fields with many rows in the CAN Scrapers state provider to `data/can-scrapers-state-providers/timeseries-common.csv`

## Tested

`git pull` in -public and -model repo to get the latest in the main branch. Than ran `python ./scripts/update_can_scraper_state_providers.py --no-fetch` to incorporate changes in this branch without changing any upstream data. This grew the output from 4606 lines/143kB to 85020 lines/3.784MB.

Current main branch
```
git cat-file blob main:data/can-scrapers-state-providers/timeseries-common.csv | git lfs smudge | csvcut -c aggregate_level | sort |uniq -c
      1 aggregate_level
   4600 county
      5 state
```

This PR branch
```
csvcut -c aggregate_level data/can-scrapers-state-providers/timeseries-common.csv  | sort |uniq -c
      1 aggregate_level
  84145 county
    874 state
```

Then added the fields in combined data (https://github.com/covid-projections/covid-data-model/pull/905) and ran `./run.py data update` and finally looked at `git difftool --no-prompt -t vimdiff main -- data/multiregion-wide-dates.csv` to see metrics added for a bunch of counties and CBSA (no states) but no metrics changed.

### CODE REVIEW

**Review by ghop02 — APPROVED** [review]
This looks good.  One temporary change I would like to make to this script is to only pull in county level data, until further improvements downstream. Thoughts?

**Comment by ghop02** [comment]
> This looks good. One temporary change I would like to make to this script is to only pull in county level data, until further improvements downstream. Thoughts?

Ah, I see that you added this as the lowest priority for all data which actually seems fine for now!

**Comment by TomGoBravo** [comment]
> > This looks good. One temporary change I would like to make to this script is to only pull in county level data, until further improvements downstream. Thoughts?
> 
> Ah, I see that you added this as the lowest priority for all data which actually seems fine for now!

File: scripts/update_can_scraper_state_providers.py [hunk 1]
```diff
@@ -54,14 +54,62 @@ def transform(dataset: ccd_helpers.CovidCountyDataset):
...
+            unit="specimens",  # Ignores less common unit=test_encounters and unit=unique_people
+            provider="state",
+            common_field=CommonFields.TOTAL_TESTS_VIRAL,
+        ),
+        ccd_helpers.ScraperVariable(
+            variable_name="pcr_tests_positive",
+            measurement="cumulative",
+            unit="specimens",  # Ignores test_encounters and unique_people
+            provider="state",
+            common_field=CommonFields.POSITIVE_CASES_VIRAL,
```
Comment (ghop02) [inline scripts/update_can_scraper_state_providers.py:111]: I actually think this should be POSITIVE_TESTS_VIRAL

Comment (TomGoBravo) [reply scripts/update_can_scraper_state_providers.py:111]: Good catch! Fix coming in 6aec8d64.

**Pull request merged**